src/views/Feed.vue

```diff
@@ -1,12 +1,11 @@
 <template>
   <ion-page>
+    <ion-button class="edit-button" v-if="canPost" @click="createPost(canPostInTeams)" shape="round" size="small">
```

Comment:
Any particular reason are using regular button rather than the [ionic fab facilities](https://ionicframework.com/docs/api/fab)?